### PR TITLE
Nutrition: Upgrade the API to v2 (For evaluation)

### PR DESCRIPTION
**Pull request by pjhampton**
## Description of new Instant Answer, or changes

Upgrades the Nutrition API. 

screen shot 2017-03-23 at 4 29 51 pm
screen shot 2017-03-23 at 4 29 41 pm

## Related Issues and Discussions

Fixes: https://github.com/duckduckgo/zeroclickinfo-spice/issues/3196

---

Instant Answer Page: https://duck.co/ia/view/nutrition

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
Deploying to Beta...

File: lib/DDG/Spice/Nutrition.pm [hunk 1]
```diff
@@ -8,24 +8,21 @@ my $attribute_regex = qr/(?^:(?^:(?:c(?:a(?:l(?:ories(?: from fat)?|cium|s)|rb(?
...
+spice headers => {
+    "Content-Type" => "application/x-www-form-urlencoded",
+    "x-app-id" => "$ENV{DDG_SPICE_NUTRITIONIX_APPID}",
+    "x-app-key" => "$ENV{DDG_SPICE_NUTRITIONIX_APIKEY}"
+};
+spice post_body => 'query=$1';
 
-    return;
+handle query => sub {
+    # return the full query
```
Comment (pjhampton) [inline lib/DDG/Spice/Nutrition.pm]: I'm not so sure iff that this is the problem. The example query here `calories in` is the measurement so `calories in banana` is like saying `how many calories in one calorie of a banana`. It's naive, but that's how the API handles that phrase 😕 

https://www.nutritionix.com/natural-demo?q=calories%20in%20banana&s=1

In my last commit, I have addressed this by ignoring the triggers and allow the API to infer the unit of measurement.

**Comment by pjhampton** [comment]
This needs to be redeployed to beta 👍

**Comment by pjhampton** [comment]
Restarting build...

**Comment by moollaza** [comment]
Closing as this isn't needed any more 👍

**Pull request closed without merging**